dirWatcher.js

```diff
@@ -0,0 +1,28 @@
+const { EventEmitter } = require('events');
+const fs = require('fs');
+
+export default class DirWatcher extends EventEmitter {
+    watch(path, delay) {
+        let content = {};
+        setInterval(() => {
+            fs.readdir(path, (err, files) => {
+                const savedPathes = Object.keys(content);
+                if (err) {
+                    console.error(err);
+                }
+                if (
+                    (!files.every(f => savedPathes.includes(f)) &&
```

Comment:
Too long condition, move to separate variable.
Also, would be nice to support files removal. And better to compare date of file change instead of size comparison.